src/pages/AddLiquidityV2/index.tsx

```diff
@@ -777,7 +780,8 @@ export default function AddLiquidity({
                   value={formattedAmounts[Field.CURRENCY_A]}
                   onUserInput={onFieldAInput}
                   hideInput={true}
-                  showMaxButton={false}
+                  onMax={null}
+                  onHalf={null}
```

Comment:
I think we should make it to optional params, so we don't have to pass null like this.
<img width="148" alt="image" src="https://user-images.githubusercontent.com/33005392/201396108-508ce40c-8e7c-4b34-8717-e14a697ceb7d.png">

Reply:
There are pros and cons for implicit and explicit passing empty value. But I prefer explicit everything possible since it make codebase much more easily to maintain, not have to guessing is this field by mistake or intented empty.